site/package.json

```diff
@@ -28,12 +28,19 @@
     "twig": "1.15.4"
   },
   "devDependencies": {
+    "@babel/core": "^7.14.2",
```

Comment:
babel is added so that we can use the ES6 module directives in the test files. We could forgo it, and then use the old-style `const { buildCourseUrl } = require('../')` type syntax, but I wanted to keep the test files written in a similar fashion as the code it's testing for developer simplicity.